src/components/Avatar/index.tsx

```diff
@@ -73,16 +76,21 @@ const Avatar: FunctionComponent<AvatarProps> = ({
         textBgColor={textBgColor}
         textColor={textColor}
         textSize={textSize}
+        alt={lock ? 'locked' : text}
       >
         {lock ? <Icon name="lock" color="gray550" /> : formatTextToAvatar(text)}
       </StyledDiv>
     ) : (
-      <StyledImg src={image} alt="" />
+      <StyledImg src={image} alt={alt} />
```

Comment:
I don't really understand why we need to add an alt here ? 🤔 

Imo this is a decorative image no ? https://www.w3.org/WAI/tutorials/images/decorative/

cc @iManu

Reply:
`alt` is mandatory on image, even if it has no content (empty alt)

https://www.w3.org/WAI/tutorials/images/

Reply:
@iManu agree, what I meant was why can't we keep it empty ? 

In the link you provided it says 

```
Decorative images: Provide a null text alternative (alt="") when the only purpose of an image is to add visual decoration to the page, rather than to convey information that is important to understanding the page.
```

Reply:
Reverted to an empty string as suggested @chambo-e